Simple patch apply after sed'ing cliffy from 0.170.0 -> 0.177.0
perhaps like
find -name "*.ts" -exec sed -i "s/std@0.170.0/std@${DENO_STD_VER}/" {} \;
--- a/dev_deps.ts
+++ b/dev_deps.ts
@@ -21,7 +21,7 @@
 export { dirname } from "https://deno.land/std@0.177.0/path/mod.ts";
 export { expandGlob } from "https://deno.land/std@0.177.0/fs/mod.ts";
 export type { WalkEntry } from "https://deno.land/std@0.177.0/fs/mod.ts";
-export { copy } from "https://deno.land/std@0.177.0/streams/conversion.ts";
+export { copy } from "https://deno.land/std@0.177.0/streams/write_all.ts";
 
 /* 3rd party */
 export { gt, lt } from "https://deno.land/x/semver@v1.4.1/mod.ts";
